netket/optimizer/qgt/qgt_jacobian_dense.py

```diff
@@ -49,6 +55,18 @@ def QGTJacobianDense(
     The details on how the ⟨S⟩⁻¹⟨F⟩ system is solved are contained in
     the field `sr`.
 
+    Numerical estimates of the QGT are usually ill-conditioned and require
+    regularisation. The standard approach is to add a positive constant to the diagonal;
+    alternatively, Becca and Sorella (2017) propose scaling this offset with the
+    diagonal entry itself. NetKet allows using both in tandem:
+
+    .. math::
+
+        S_{ii} \\mapsto S_{ii} + \\epsilon_1 S_{ii} + \\epsilon_2;
+
+    :math:`\\epsilon_{1,2}` are specified using `diag_scale` and `diag_shift`,
+    respectively. By default, `diag_scale=0.0`, `diag_shift=0.01`.
```

Comment:
```suggestion
    :math:`\\epsilon_{1,2}` are specified using `diag_scale` and `diag_shift`,
    respectively.
```

those are only the defaults for SR, not the real defaults of this